tools/darwin/packaging/ios/mkdeb-ios.sh.in

```diff
@@ -75,22 +81,26 @@ rm -rf $DIRNAME/$ARCHIVE
 mkdir -p $DIRNAME/$PACKAGE/DEBIAN
 if $ARM64; then
   echo "Package: $PACKAGE_ARM64"                  >  $DIRNAME/$PACKAGE/DEBIAN/control
-  echo "Name: @APP_NAME@-iOS (64-bit)"            >> $DIRNAME/$PACKAGE/DEBIAN/control
+  echo "Name: @APP_NAME@-${PP_DEVICE} (64-bit)"            >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Pre-Depends: cy+cpu.arm64"                >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Conflicts: $PACKAGE"                      >> $DIRNAME/$PACKAGE/DEBIAN/control
   echo "Replaces: $PACKAGE"                       >> $DIRNAME/$PACKAGE/DEBIAN/control
 else
   echo "Package: $PACKAGE"                        >  $DIRNAME/$PACKAGE/DEBIAN/control
-  echo "Name: @APP_NAME@-iOS"                     >> $DIRNAME/$PACKAGE/DEBIAN/control
+  echo "Name: @APP_NAME@-${PP_DEVICE}"                     >> $DIRNAME/$PACKAGE/DEBIAN/control
 fi
 echo "Priority: Extra"                            >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Depends: firmware (>= 9.0)"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+if [ "@PLATFORM@" == "appletvos" ]; then
+  echo "Depends: firmware (>= @CMAKE_XCODE_ATTRIBUTE_TVOS_DEPLOYMENT_TARGET@)" >> $DIRNAME/$PACKAGE/DEBIAN/
+else
+  echo "Depends: firmware (>= @CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET@)"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+fi
 echo "Version: $VERSION-$REVISION"                >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Architecture: iphoneos-arm"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+echo "Architecture: @PLATFORM@-arm"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
 echo "Installed-Size: $SIZE"                      >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Description: @APP_NAME@ Entertainment Center for iOS" >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Homepage: http://kodi.tv/"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Maintainer: Memphiz"                        >> $DIRNAME/$PACKAGE/DEBIAN/control
+echo "Description: @APP_NAME@ Entertainment Center for ${PP_DEVICE}" >> $DIRNAME/$PACKAGE/DEBIAN/control
+echo "Homepage: https://kodi.tv/"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
```

Comment:
use `@APP_WEBSITE@` and change it in a separate commit (can be the same as the above one)